src/common/reducers/userSummaries.js

```diff
@@ -40,22 +43,27 @@ export default function userSummaries(state = initialState, action) {
         isBusy: true,
       })
     case DEACTIVATE_USER_SUCCESS:
-      {
-        const userAttrs = action.response || {}
-        const userSummary = state.userSummaries[userAttrs.id] || {}
-        const user = Object.assign({}, userSummary.user, userAttrs)
-        const newUserSummary = Object.assign({}, userSummary, {user})
-        const userSummaries = Object.assign({}, state.userSummaries, {[user.id]: newUserSummary})
-        return Object.assign({}, state, {
-          isBusy: false,
-          userSummaries,
-        })
-      }
+      return Object.assign({}, state, {
+        isBusy: false,
+      })
     case DEACTIVATE_USER_FAILURE:
       return Object.assign({}, state, {
         isBusy: false,
       })
 
+    case REACTIVATE_USER_REQUEST:
+      return Object.assign({}, state, {
+        isBusy: true,
+      })
+    case REACTIVATE_USER_SUCCESS:
+      return Object.assign({}, state, {
+        isBusy: false,
+      })
+    case REACTIVATE_USER_FAILURE:
+      return Object.assign({}, state, {
+        isBusy: false,
+      })
```

Comment:
Protip: if you want the same behavior for two different cases in a switch/case, you can just combine them like so:

```javascript
  case REACTIVATE_USER_SUCCESS:
  case REACTIVATE_USER_FAILURE:
    return Object.assign({}, state, {
      isBusy: false,
    })
```